### PR TITLE
adding caritonaranjo

### PR DESCRIPTION
**Pull request by caritonaranjo**
cambiando el nombre a la rama, y agregando mi nombre al teem

### CODE REVIEW

**Review by mfbarrera — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -19,4 +19,5 @@ All team members should do the following:
 ## Team members
 | Name | Profile            |
 | ---- | ------------------ |
-| Name |[Username](github.link) |
\ No newline at end of file
+| Name |[Username](github.link) |
```
Comment (omeneses) [inline README.md:22]: This line needs to be deleted

File: README.md [hunk 2]
```diff
@@ -19,4 +19,5 @@ All team members should do the following:
...
+| Carolina Naranjo |[caritonaranjo](https://github.com/caritonaranjo) |
```
Comment (omeneses) [inline README.md]: This code should be in line 22.   Code in line 22 should be removed.

**Review by caritonaranjo — COMMENTED** [review]
ya realice los cambios

File: README.md (hunk 2 above)
Comment (omeneses) [reply README.md]: Nice work!!!

**Review by omeneses — DISMISSED** [review]

**Review by mfbarrera — DISMISSED** [review]

**Review by omeneses — APPROVED** [review]

**Review by sjaramilloo — APPROVED** [review]

**Pull request merged**